docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
+
+## Syntax
+
+Executes parameterless function:
```

Comment:
If I was a user, I would wonder what function you're talking about here. We need to explain somewhere that this is a user-defined function and where that function lives.

Reply:
Added an explanation above